src/components/ElvaYBPractice2/MainComponent.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+import { Slider, Switch } from 'antd'
+
+import IconCheck from '../../assets/ElvaYongbin/images/icon-check.svg'
+import SliderIcon from '../../../../assets/ElvaYongbin/images/icon-slider.svg'
+
+import {
+  MainComponent,
+  PageViewContainer,
+  PageViewPriceContainer,
+  PriceMonthContainer,
+  PriceContainer,
+  MonthContainer,
+  Slidercontainer,
+  StyleSlider,
+  ToggleContainer,
+  ToggleContainerText,
+  DiscountButton2,
+  DiscountButton1,
+  MainBottomComponent,
+  BottomLeftContainer,
+  BottomLeftText1,
+  BottomLeftText2,
+  BottomLeftText3,
+  TrialButtonContainer,
+  CheckIcon,
+  StartTrialButton,
+} from './styles/MainComponent.styled'
+const marks = {
+  0: '',
+  25: '',
+  50: '',
+  75: '',
+  100: '',
+}
+
+const MainComponentContainer = () => {
+  const [views, setViews] = useState(50)
+  const [toggle, setToggle] = useState(false)
+  return (
+    <MainComponent>
+      <PageViewPriceContainer>
+        {toggle === false ? (
+          views === 0 ? (
+            <PageViewContainer>{10}k PAGEVIEWS</PageViewContainer>
+          ) : views === 25 ? (
+            <PageViewContainer>{50}k PAGEVIEWS</PageViewContainer>
+          ) : views === 50 ? (
+            <PageViewContainer>{100}k PAGEVIEWS</PageViewContainer>
+          ) : views === 75 ? (
+            <PageViewContainer>{500}k PAGEVIEWS</PageViewContainer>
+          ) : (
+            <PageViewContainer>{1}M PAGEVIEWS</PageViewContainer>
+          )
+        ) : views === 0 ? (
+          <PageViewContainer>{10}k PAGEVIEWS</PageViewContainer>
+        ) : views === 25 ? (
+          <PageViewContainer>{50}k PAGEVIEWS</PageViewContainer>
+        ) : views === 50 ? (
+          <PageViewContainer>{100}k PAGEVIEWS</PageViewContainer>
+        ) : views === 75 ? (
+          <PageViewContainer>{500}k PAGEVIEWS</PageViewContainer>
+        ) : (
+          <PageViewContainer>{1}M PAGEVIEWS</PageViewContainer>
+        )}
+        <PriceMonthContainer>
+          {toggle === false ? (
+            views === 0 ? (
+              <PriceContainer>$8.00</PriceContainer>
+            ) : views === 25 ? (
+              <PriceContainer>$12.00</PriceContainer>
+            ) : views === 50 ? (
+              <PriceContainer>$16.00</PriceContainer>
+            ) : views === 75 ? (
+              <PriceContainer>$24.00</PriceContainer>
+            ) : (
+              <PriceContainer>$36.00</PriceContainer>
+            )
+          ) : views === 0 ? (
+            <PriceContainer>$6.00</PriceContainer>
+          ) : views === 25 ? (
+            <PriceContainer>$9.00</PriceContainer>
+          ) : views === 50 ? (
+            <PriceContainer>$12.00</PriceContainer>
+          ) : views === 75 ? (
+            <PriceContainer>$18.00</PriceContainer>
+          ) : (
+            <PriceContainer>$27.00</PriceContainer>
+          )}
+          <MonthContainer>/month</MonthContainer>
+        </PriceMonthContainer>
```

Comment:
Consider abstracting Price Month Container where u pass in the boolean value of toggle as a props. You can add useState hooks inside that component to change the content being displayed.